crossplane.yaml

```diff
@@ -47,47 +47,9 @@ metadata:
 spec:
   crossplane:
-    version: ">=v1.0.0-0"
```

Comment:
nice thoroughness to bump the min crossplane version to what will support the features we are using in the compositions! 💪